mpd/mpd_read_write_test.go

```diff
@@ -6,140 +6,114 @@ import (
 	"testing"
 	"time"
 
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/suite"
+	"github.com/stretchr/testify/require"
 	"github.com/zencoder/go-dash/helpers/testfixtures"
 )
 
-type MPDReadWriteSuite struct {
-	suite.Suite
+func TestReadingManifests(t *testing.T) {
```

Comment:
This was the only actual change - I bundled a few different tests into this one loop

Reply:
Fine by me.